packages/grid/src/GridColumn.tsx

```diff
@@ -54,6 +54,20 @@ export interface GridColumnProps<T = any> {
    * Unique identifier of the column.
    * */
   id: string; // TODO make optional
+  /**
+   * Enables sorting for the column.
+   * To enable column header's keyboard navigation on sort,
+   * users need to set `headerIsFocusable` prop to `true` in Grid component.
+   * */
+  isSortable?: boolean;
+  /**
+   * Custom sorting function.
+   * */
+  customSort?: (args: {
+    rowData: T[];
+    sortByColumnId: string;
```

Comment:
I don't think this is necessary. If this is a prop for the GridColumn, then it doesn't make sense because this function will only get called on this same column and therefore always be the same.

Reply:
true that!